samples/DotNet/Microsoft.Azure.ServiceBus/BasicSendReceiveUsingQueueClient/Program.cs

```diff
@@ -26,20 +28,24 @@ static void Main(string[] args)
         {
             const int numberOfMessages = 10;
             queueClient = new QueueClient(ServiceBusConnectionString, QueueName);
+            deadLetterReceiver = new MessageReceiver(ServiceBusConnectionString, EntityNameHelper.FormatDeadLetterPath(QueueName), ReceiveMode.PeekLock);
```

Comment:
Given the sample is focusing on "basic send and receive using `QueueClient`", I'm not convince mixing `MessageReceiver` here is the right approach.